--- a/src/Plugin/views/argument/DateRange.php
+++ b/src/Plugin/views/argument/DateRange.php
@@ -22,7 +22,7 @@
   /**
    * Overrides Drupal\views\Plugin\views\argument\Formula::init().
    */
-  public function init(ViewExecutable $view, DisplayPluginBase $display, array &$options = NULL) {
+  public function init(ViewExecutable $view, DisplayPluginBase $display, ?array &$options = NULL) {
     parent::init($view, $display, $options);

     switch ($options['id']) {
