--- a/src/Plugin/views/field/Combine.php
+++ b/src/Plugin/views/field/Combine.php
@@ -25,7 +25,7 @@
   /**
    * {@inheritdoc}
    */
-  public function init(ViewExecutable $view, DisplayPluginBase $display, array &$options = NULL) {
+  public function init(ViewExecutable $view, DisplayPluginBase $display, ?array &$options = NULL) {
     parent::init($view, $display, $options);
     $this->options['exclude'] = 1;
   }
--- a/src/Plugin/views/filter/Combine.php
+++ b/src/Plugin/views/filter/Combine.php
@@ -24,7 +24,7 @@
   /**
    * {@inheritdoc}
    */
-  public function init(ViewExecutable $view, DisplayPluginBase $display, array &$options = NULL) {
+  public function init(ViewExecutable $view, DisplayPluginBase $display, ?array &$options = NULL) {
     parent::init($view, $display, $options);
     $this->valueTitle = $this->t('Views');
   }
--- a/src/ViewsCombiner.php
+++ b/src/ViewsCombiner.php
@@ -212,7 +212,7 @@
    * @return \Drupal\views\ViewExecutable|null
    *   Returns the built view executable.
    */
-  public function buildView(string $view_id, string $display_id, ViewExecutable $base_view, FieldPluginBase $field_handler = NULL) {
+  public function buildView(string $view_id, string $display_id, ViewExecutable $base_view, ?FieldPluginBase $field_handler = NULL) {
     if ($view = $this->getView($view_id, $display_id)) {
       $input = $base_view->exposed_raw_input;

